src/gwt/src/org/rstudio/studio/client/application/StudioClientApplicationConstants_fr.properties

```diff
@@ -154,3 +154,4 @@ saveNoLabel=Ne pas sauvegarder
 consolePaneName=Console
 activeText=(active)
 requestLogVisualization=<p>Cliquez sur une demande pour voir les détails. Cliquez sur l''arrière-plan pour afficher à nouveau ces instructions.</p><h4>Commandes disponibles:</h4><ul><li>Esc: Fermer</li><li>P: Lecture/pause</li><li>E: Exporter</li><li>I: Importer</li><li>+/-: Zoom avant/arrière</li></ul>
+visitWebsiteForNewVersionText=Veuillez visiter https://posit.co/download/rstudio-desktop/ pour vérifier si une nouvelle version est disponible.
```

Comment:
@romainfrancois I used a translator to generate this -- could you please confirm if it's correct?

Reply:
✅ 🇫🇷